eng/performance/maui_scenarios_ios.proj

```diff
@@ -151,16 +149,17 @@
           sign MauiBlazoriOSDefault.app
 
           # Testing commands
-          $(Python) test.py devicestartup --device-type ios --package-path MauiBlazoriOSDefault.app --package-name net.dot.mauiblazortesting --scenario-name "%(Identity)" --use-fully-drawn-time --fully-drawn-magic-string __MAUI_Blazor_WebView_OnAfterRender__ --startup-iterations 5
+          $(Python) test.py devicestartup -1-device-type ios -1-package-path MauiBlazoriOSDefault.app -1-package-name net.dot.mauiblazortesting -1-scenario-name "%(Identity)" -1-use-fully-drawn-time -1-fully-drawn-magic-string __MAUI_Blazor_WebView_OnAfterRender__ -1-startup-iterations 5
```

Comment:
I know nothing about python, what does this change do?
```diff
-- --device-type
++ -1-device-type
```

Reply:
It is not clear in the compare, but this is part of a larger comment out block and the xml gets unhappy when there is --'s inside a larger block comment. So, this is just something to stop the comment from causing errors/warnings.